src/irmin-pack/unix/mapping_file_intf.ml

```diff
@@ -50,6 +50,14 @@ module type S = sig
       Creates temporary files in [root] that are unlinked before the function
       returns. *)
 
+  val create_rev :
```

Comment:
If we go with this code, lets rename this to `create` and remove the existing code.

Reply:
I still think this. :smile: